Jint/Engine.cs

```diff
@@ -162,6 +167,39 @@ public Engine(Action<Options> options)
             ClrTypeConverter = new DefaultTypeConverter(this);
             BreakPoints = new List<BreakPoint>();
             DebugHandler = new DebugHandler(this);
+
+            if (Options._ExtensionMethodsTypes.Any())
```

Comment:
Concurrent Dictionary is not available in PCL...
Do you think it's necessary to use a lock for keeping the Dictionary thread-safe?
Or is it fine with just a static Dictionary?